.github/CODEOWNERS

```diff
@@ -6,7 +6,7 @@
 # Note: This file is autogenerated. To make changes to the codeowner team, please update .repo-metadata.json.
 
 # @googleapis/yoshi-python is the default owner for changes in this repo
-*     @googleapis/yoshi-python
+*     @googleapis/yoshi-python @arithmetic1728 @sai-sunder-s @TimurSadykov
```

Comment:
Is there a team that can be assigned, rather than individuals?

Reply:
We do not have one yet. We will create one and I will update this PR